src/SFA.DAS.ApplyService.Database/Tables/AssessorPageReviewOutcome.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE TABLE [dbo].[AssessorPageReviewOutcome](
+	[Id] [uniqueidentifier] NOT NULL,
+	[ApplicationId] [uniqueidentifier] NOT NULL,
+	[SequenceNumber] [int] NOT NULL,
+	[SectionNumber] [int] NOT NULL,
+	[PageId] [nvarchar](50) NOT NULL,
+	[Assessor1UserId] NVARCHAR(256) NULL,     
+	[Assessor1ReviewStatus] NVARCHAR(20) NULL, 
+    [Assessor1ReviewComment] NVARCHAR(MAX) NULL,
+	[Assessor2UserId] NVARCHAR(256) NULL, 
+	[Assessor2ReviewStatus] NVARCHAR(20) NULL,
+	[Assessor2ReviewComment] NVARCHAR(MAX) NULL,
+	[CreatedAt] DATETIME2 NOT NULL, 
+    [CreatedBy] NVARCHAR(256) NOT NULL, 
+    [UpdatedAt] DATETIME2 NULL, 
+    [UpdatedBy] NVARCHAR(256) NULL, 
+ CONSTRAINT [PK_AssessorPageReviewOutcome] PRIMARY KEY CLUSTERED 
+(
+	[Id] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY], 
+) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
+GO
+
+ALTER TABLE [dbo].AssessorPageReviewOutcome ADD  DEFAULT (newid()) FOR [Id]
+GO
```

Comment:
We'll need some indexes.  Application Id as a minimum.

Reply:
The querying of the data will be by:
GetAllStatuses()
-	ApplicationId, (Assessor1UserId or Assessor2UserId)
GetStatusesPerSection() – to work out SectionStatus (might not be needed if use above and group them) 
-	ApplicationId, SequenceNumber, SectionNumber (Assessor1UserId or Assessor2UserId)
GetPageStatus()
-	ApplicationId, PageId, (Assessor1UserId or Assessor2UserId)

So, which fields should I index?

Reply:
I think an index on ApplicationId will be sufficient.

Reply:
index IX_AssessorPageReviewOutcome_ApplicationId added